packages/rules/src/mustSatisfyPeerDependencies.ts

```diff
@@ -16,6 +16,7 @@ const Options = r.Union(
     skipUnparseableRanges: r.Undefined,
     dependencyWhitelist: r.Undefined,
     dependencyBlacklist: r.Undefined,
+    enforceForDevDependencies: r.Undefined,
```

Comment:
I'm not certain about this name, especially since I'm thinking we'll one day want something like `mustForwardPeerDeps: isBundle: (packageJson: PackageJson) => boolean` which would require webpack bundles to take dependencies on inherited peer deps, and all other packages to forward peer deps / take dev deps on inherited peer deps.

Reply:
i think it's fine to start with this for now, since the name describes exactly what we're trying to do right now